app_vue/src/pages/MapPage.vue

```diff
@@ -37,11 +34,7 @@
       
       <NavigationBar></NavigationBar>
       <SensorSidebar></SensorSidebar>
-      <SensorMap
-        :alertThreshold="alertThreshold"
-        :filterThresholdMaximum="filterThresholdMaximum"
-        :filterThresholdMinimum="filterThresholdMinimum">
```

Comment:
these props were used to determine the bin icons, removing as they are no longer in use due to this PR